src/store/index.js

```diff
@@ -107,33 +108,43 @@ export default createStore({
         state.currentUser.mail === state.users[0].mail &&
         state.currentUser.password === state.users[0].password
       ) {
-        state.currentUser.logged = true;
-        state.currentUser.loggedUserId = 1;
-        state.currentUser.mail = "";
-        state.currentUser.password = "";
+        const { currentUser } = state;
+        currentUser.logged = true;
+        currentUser.loggedUserId = 1;
+        currentUser.mail = "";
+        currentUser.password = "";
         state.textInfo = "";
       } else {
         state.textInfo = "Invalid e-mail or password";
       }
     },
     addToCart(state, payload) {
-      state.shoppingCart.cartId = 1;
-      state.shoppingCart.userId = state.currentUser.loggedUserId;
-      state.shoppingCart.productsInStock.id = 1;
-      state.shoppingCart.productsInStock.quantity += 1;
-      state.shoppingCart.productsInStock.full = true;
+      const { currentUser, users, textInfo, products, shoppingCart } = state;
+      shoppingCart[0].full = true;
     },
   },
   actions: {
-    login(state) {
+    LOGIN_USER(state) {
       setTimeout(() => {
         this.commit("login");
       }, 1000);
     },
   },
   getters: {
     totalPrice(state) {
-      return state.shoppingCart.productsInStock.quantity * 20;
+      return state.shoppingCart[0].products[0].points;
+    },
+    displayUserPoints(state) {
+      return state.shoppingCart[0].userPoints;
+    },
+    pointsAfterShopping(state) {
+      return (
+        state.shoppingCart[0].userPoints -
+        state.shoppingCart[0].products[0].points
+      );
+    },
+    productInCart(state) {
+      return state.shoppingCart[0].products[0].name;
     },
```

Comment:
Jeżeli chcesz dostać tylko jeden przedmiot z koszyka to funkcja powinna przyjmować id, które dostanie od komponentu - czyli albo kminisz jak tut dodać id albo do mutacji

I nie tak na. sztywno tylko:
- albo filter -> jak chcesz zrobić tablice
- albo find -> jeżeli chcesz jeden obiekt